localenv/README.md

```diff
@@ -139,6 +139,28 @@ Note that you have to go through an additional "login" step by providing you IPv
 
 After stopping the script it is necessary to clear the environment using the command described in [Shutting down](#Shutting-down). This is necessary as on a new run of the scripts (with autopeering or not) the wallet address url will differ.
 
+### Debugging
```

Comment:
Maybe we can add a section with the launch.json configuration for VSCode as well?

Also, did you want to keep it just in this readme, or should we also add it to the documentation site?

Reply:
added to the documentation site: https://github.com/interledger/rafiki/pull/2718/commits/02a5e6b780037021ec2fe83ffab4c4fe4191e2d3